user_api/serializers.py

```diff
@@ -8,20 +8,32 @@ class UserSerializer(serializers.ModelSerializer):
 
     class Meta:
         model=User
-        exclude=['password','active','staff','admin','last_login']
+        fields=['id','email','name','city','state','zipcode','balance','phone','created_at','updated_at']
     
     def create(self,validate_data):
         return User.objects.create(**validate_data)
 
     def validate_zipcode(self,value):
-        if len(str(value)) == 6:
-            return value
-        return serializers.ValidationError("Invalid ZipCode")
+        if len(str(value)) != 6:
```

Comment:
I have asked to create a constant for 6 in a seperate file and use it over here. Why is it not created ? Also what if the zipcode is 000000 is it  a valid one ?